common/data_guide.md

```diff
@@ -46,7 +34,7 @@ BCDA follows the bulk FHIR specification, and provides the same data as the CCLF
 * The **Coverage** file provides information about the beneficiaries’ insurance coverage, including information about dual coverage.
 
 ## BCDA Data Dictionary
-The CCLF to [BCDA Data Dictionary](assets/data/CCLF_BCDA_BB_Crosswalk.xlsx) is provided to BCDA users to facilitate their data mapping process. Because BCDA delivers data in the FHIR format, data field names will be different from previous data fields delivered through CCLFs. In this spreadsheet, BCDA has provided a crosswalk between all CCLF data fields and their new locations within BCDA Claims files*. The Data Dictionary also provides supplementary context for each of the CCLF/BCDA data fields, including:
+The CCLF to [BCDA Data Dictionary](/assets/data/CCLF_BCDA_BB_Crosswalk.xlsx) is provided to BCDA users to facilitate their data mapping process. Because BCDA delivers data in the FHIR format, data field names will be different from previous data fields delivered through CCLFs. In this spreadsheet, BCDA has provided a crosswalk between all CCLF data fields and their new locations within BCDA Claims files*. The Data Dictionary also provides supplementary context for each of the CCLF/BCDA data fields, including:
```

Comment:
Request to pull the link all the way back to "CCLF to" in the first line of this graph. (ie The [CCLF to BCDA Data Dictionary](/assets/data/CCLF_BCDA_BB_Crosswalk.xlsx) ...)